cmd/config/metrics-profiles/metrics.yml

```diff
@@ -187,3 +187,13 @@
 
 - query: rate ( node_vmstat_pgmajfault[1m] )
   metricName: nodeMajorFaults
+
+# cgroup CPU instantaneous per-second rate of increase over the last 2 minutes
+- query: sum (  irate( container_cpu_usage_seconds_total { id =~ "/system.slice|/system.slice/kubelet.service|/system.slice/ovs-vswitchd.service|/system.slice/crio.service|/kubepods.slice" }[2m])  )  by   (   id , instance )
+  metricName: cgroupCPU
+
+# RSS Memory 
+- query: sum( container_memory_rss { id =~ "/system.slice|/system.slice/kubelet.service|/system.slice/ovs-vswitchd.service|/system.slice/crio.service|/kubepods.slice"}) by (id)
+  metricName: cgroupMemoryRSS
+  instant: true
+  captureStart: true
```

Comment:
Would remove `captureStart` and `instant` so you can capture the time-series resolution. We are already capturing start and end values for thise metric in an above query.